launch_ros/launch_ros/actions/load_composable_nodes.py

```diff
@@ -82,6 +86,21 @@ def __init__(
         self.__final_target_container_name = None  # type: Optional[Text]
         self.__logger = launch.logging.get_logger(__name__)
 
+    @classmethod
+    def parse(cls, entity: Entity, parser: Parser):
+        """Parse load_composable_node."""
+        _, kwargs = super().parse(entity, parser)
+
+        kwargs['target_container'] = parser.parse_substitution(
+            entity.get_attr('target', data_type=str))
+
+        composable_nodes = entity.get_attr('composable_node', data_type=List[Entity])
+        if composable_nodes is not None:
```

Comment:
@kenji-miyake hmm, I think you don't need to check for `None`. It's not optional.

Reply:
Exactly, removed in cb67451.